src/common/BackTrace.h

```diff
@@ -18,7 +18,7 @@ struct BackTrace {
   size_t size;
   char **strings;
 
-  explicit BackTrace(int s) : skip(s) {
+  explicit BackTrace(int s) : skip(s), array{} {
```

Comment:
@tchaikov Done. I think this applies now: http://en.cppreference.com/w/cpp/language/zero_initialization

`     If T is array type, each element is zero-initialized `

I will squash it once we finalize the change.